initramfs-init.in

```diff
@@ -269,7 +269,7 @@ set -- $(cat /proc/cmdline)
 myopts="alpine_dev autodetect autoraid chart cryptroot cryptdm cryptheader cryptoffset
 	cryptdiscards debug_init dma init_args keep_apk_new modules ovl_dev pkgs quiet
 	root_size root usbdelay ip alpine_repo apkovl alpine_start splash blacklist
-	overlaytmpfs rootfstype rootflags nbd resume s390x_net dasd"
+	overlaytmpfs rootfstype rootflags nbd resume s390x_net dasd nameserver ssh_install ssh_key ssh_pass"
```

Comment:
I think we don't need `nameserver` which can/should be passed via `ip`, similar to how it is done for [nfsroot](https://www.kernel.org/doc/Documentation/filesystems/nfs/nfsroot.txt). I also don't think we need the `ssh_install`. We can enable ssh server if either `ssh_key` or `ssh_pass` is set.

Reply:
Fixed.